en/sql/schema/stored_routine_stmt.rst

```diff
@@ -0,0 +1,366 @@
+
+:meta-keywords: procedure definition, create procedure, drop procedure, function definition, create function, drop function
+:meta-description: Define functions/procedures in CUBRID database using create procedure, create function, drop procedure and drop function statements.
+
+
+************************************************
+STORED FUNCTION/PROCEDURE DEFINITION STATEMENTS
+************************************************
+
+.. _create-procedure:
+
+CREATE PROCEDURE
+=================
+
+Create stored procedure using the **CREATE PROCEDURE** statement.
+The other languages except Java do not support stored procedure. 
+In CUBRID, only Java can implement stored procedure.
+See details of how to use Stored Procedure, please refer to the :doc:`/sql/jsp`.
+
+::
+
+    CREATE [OR REPLACE] PROCEDURE procedure_name [(<parameter_definition> [, <parameter_definition>] ...)]
+    {IS | AS} LANGUAGE JAVA <java_call_specification>
+    COMMENT 'sp_comment_string';
+    
+        <parameter_definition> ::= parameter_name [IN|OUT|IN OUT|INOUT] sql_type [COMMENT 'param_comment_string']
+        <java_call_specification> ::= NAME 'java_method_name (java_type [,java_type]...) [return java_type]'
+
+*   *procedure_name*: Specifies the name of the stored procedure(maximum 254 bytes).
+*   *parameter_name*: Specifies the name of the parameter (maximum 254 bytes).
+*   *sql_type*: Specifies the data type of the parameter. See details on the data types that can be used for the parameter, refer to the :ref:`jsp-type-mapping`.
+*   *param_comment_string*: Specifies comment of the parameter.
+*   *sp_comment_string*: Specifies comment of the stored procedure.
+*   *java_method_name*: Specifies the name of Java method name, including the name of the class it belongs to.
+*   *java_type*: Specifies the Java data type. See details on the Java data types that can be used to return, refer to the :ref:`jsp-type-mapping`.
+
+You must publish Java classes by using Java Call Specifications (<*java_call_specification*>) becuase it is not known how a function in a class will be called by SQL statements or Java applications when Java classes are loaded.
```

Comment:
```suggestion
You must publish Java classes by using Java Call Specifications (<*java_call_specification*>) because it is unknown how a function in a class will be called by SQL statements or Java applications when Java classes are loaded.
```